blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxIO.java

```diff
@@ -0,0 +1,147 @@
+package com.rackspacecloud.blueflood.io.datastax;
+
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.exceptions.NoHostAvailableException;
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
+import com.datastax.driver.core.policies.TokenAwarePolicy;
+import com.rackspacecloud.blueflood.io.IOConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+
+/**
+ * Created by shin4590 on 3/10/16.
+ */
+public class DatastaxIO {
+    private static final DatastaxIO INSTANCE = new DatastaxIO();
+    private static final Logger LOG = LoggerFactory.getLogger(DatastaxIO.class);
+    private static final IOConfig ioconfig = IOConfig.singleton();
+
+    private static  Cluster cluster;
+    private static Session session;
+
+//    private final static Meter hostMeter = utils.Metrics.meter(DatastaxIO.class, "Hosts Connected");
+//    private final static Meter openConnectionsMeter = utils.Metrics.meter(DatastaxIO.class, "Total Open Connections");
+//    private final static Meter inFlightQueriesMeter = utils.Metrics.meter(DatastaxIO.class, "Total InFlight Queries");
+//    private final static Meter trashedConnectionsMeter = utils.Metrics.meter(DatastaxIO.class, "Total Trashed Connections");
+//    private final static Meter maxLoadMeter = utils.Metrics.meter(DatastaxIO.class, "Maximum Load");
+
+    static {
+        connect();
+        monitorConnection();
```

Comment:
I agree. I don't like the static-ness either. I was trying to keep the behavior of how these drivers/connection pools the same. Astyanax connection pools are initialized in similar manner. I first thought about making these connect() & monitorConnection() instance methods but then I think that I don't know where they should be called from -- would all the Service classes call them? Right now, it sort of happens when the AstyanaxIO class is loaded. Let's discuss.

I am spoiled from Spring. I never had to think about this when I used Spring.